src/Tizen.Applications.Cion/Tizen.Applications/PeerInfo.cs

```diff
@@ -0,0 +1,218 @@
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get device platform version.");
+                    return "";
+                }
+                return devicePlatformVersion;
+            }
+        }
+
+        /// <summary>
+        /// Gets the device type.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string DeviceType
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionPeerInfo.CionPeerInfoGetDeviceType(_handle, out string deviceType);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get device platform type.");
+                    return "";
+                }
+                return deviceType;
+            }
+        }
+
+        /// <summary>
+        /// Gets the application id.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string AppId
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionPeerInfo.CionPeerInfoGetAppId(_handle, out string AppId);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get application id.");
+                    return "";
+                }
+                return AppId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the application version.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string AppVersion
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionPeerInfo.CionPeerInfoGetAppVersion(_handle, out string AppVersion);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get application version.");
+                    return "";
+                }
+                return AppVersion;
+            }
+        }
+
+        /// <summary>
+        /// Gets the UUID.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string UUID
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionPeerInfo.CionPeerInfoGetUuid(_handle, out string uuid);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get UUID.");
+                    return "";
+                }
+                return uuid;
+            }
+        }
+
+        #region IDisposable Support
+        private bool disposedValue = false;
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
+        /// </summary>
```

Comment:
param  not described
`/// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>`

Reply:
I added missing doxygen comment.